pkg/regressionallowances/intentional_4.15_regressions.go

```diff
@@ -22,4 +24,23 @@ func init() {
 				ReasonToAllowInsteadOfFix: "",
 			})
 	*/
+
+	mustAddIntentionalRegression(
+		release415,
+		IntentionalRegression{
+			JiraComponent: "Networking / openshift-sdn",                       // Jira Component,  not team name
+			TestID:        "cluster install:0cb1bb27e418491b1ffdacab58c5c8c0", // ask TRT for the ID for your TestName
+			TestName:      "install should succeed: overall",                  // this helps approvers recognize at a glance
+			Variant: api.ComponentReportColumnIdentification{ // this indicates the selectivity of the choice
+				Network:  "sdn",
+				Upgrade:  "upgrade-micro",
+				Arch:     "amd64",
+				Platform: "metal-ipi",
+			},
+			PreviousPassPercentage:    100,
+			PreviousSampleSize:        61, // number of runs used for the percentage
+			RegressedPassPercentage:   50,
+			RegressedSampleSize:       12, // number of runs used for the percentage
```

Comment:
Are teams supposed to come up with these?  If so, we'll probably need to give guidance.